src/kit/Pivot.tsx

```diff
@@ -40,4 +40,5 @@ const Pivot: React.FC<PivotProps> = ({ type = 'primary', ...props }: PivotProps)
   return <Tabs className={css.base} type={tabType} {...props} />;
 };
 
+export type { TabsProps } from 'antd';
```

Comment:
should be able to just export the `PivotProps` we've already defined in this file

Reply:
Why are you exporting `TabsProps` from antd to begin with if it's not what the component takes?

Reply:
I'm looking at some of the usages and the correct type (like in Markdown.tsx) is the existing `TabItem` type.

Reply:
All usages I can see are being passed into the `Pivot` component which has its own prop type, as @johnkim-det noted.

Reply:
You should make the naming match the component though.